src/config.c

```diff
@@ -3069,6 +3069,7 @@ standardConfig static_configs[] = {
     createBoolConfig("replica-ignore-disk-write-errors", NULL, MODIFIABLE_CONFIG, server.repl_ignore_disk_write_error, 0, NULL, NULL),
     createBoolConfig("extended-redis-compatibility", NULL, MODIFIABLE_CONFIG, server.extended_redis_compat, 0, NULL, updateExtendedRedisCompat),
     createBoolConfig("enable-debug-assert", NULL, IMMUTABLE_CONFIG | HIDDEN_CONFIG, server.enable_debug_assert, 0, NULL, NULL),
+    createBoolConfig("io_uring", NULL, DEBUG_CONFIG | IMMUTABLE_CONFIG, server.io_uring_enabled, 0, NULL, NULL),
```

Comment:
This just occurred to me. we haven't used "\_" before in the config name but "_" is part of the "io_uring" name. So the config name looks kind of funny...
```suggestion
    createBoolConfig("enable-io_uring", NULL, DEBUG_CONFIG | IMMUTABLE_CONFIG, server.io_uring_enabled, 0, NULL, NULL),
```

Reply:
I noticed the underscore too but I didn't comment. :)

Regarding "enable", many configs have it as a suffix (aof-timestamp-enabled, syslog-enabled, cluster-enabled, crash-log-enabled, crash-memcheck-enabled) and only one use prefix (enable-debug-assert). Most other bool configs don't have it at all. Some of them include the word "use".

Reply:
Changed to `enable-io_uring`.